gax-java/gax/src/main/java/com/google/api/gax/rpc/WatchdogProvider.java

```diff
@@ -30,18 +30,24 @@
 package com.google.api.gax.rpc;
 
 import com.google.api.core.ApiClock;
+import com.google.api.core.InternalApi;
+import com.google.api.core.ObsoleteApi;
 import java.util.concurrent.ScheduledExecutorService;
 import javax.annotation.Nonnull;
-import org.threeten.bp.Duration;
 
+@InternalApi
```

Comment:
Adding InternalApi is considered a breaking change: https://github.com/googleapis/sdk-platform-java/blob/e1e1fb66d95adc08baea71c5a0607c20d5a57a8b/api-common-java/src/main/java/com/google/api/core/InternalApi.java#L45

Reply:
Removed